src/main/java/software/amazon/awssdk/crt/io/ExponentialBackoffRetryOptions.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.awssdk.crt.io;
+
+import software.amazon.awssdk.crt.CrtResource;
+
+public class ExponentialBackoffRetryOptions extends CrtResource {
```

Comment:
I feel like options is misleading.  Aren't these strategy bindings?

Reply:
These are bindings for the options structures right now.  The actual strategy bindings are intended to be added later.

https://github.com/awslabs/aws-c-io/blob/main/include/aws/io/retry_strategy.h#L101